todo_app/data/trello_items.py

```diff
@@ -109,3 +105,43 @@ def get_card_from_id(self, id):
         for card in cards:
             if int(id) == card['id']:
                 return card
+
+# Test functions
+    def create_test_board(self):
+        url = self.url + 'boards'
+        self.query['name'] = 'TEST'
```

Comment:
Hmm, my first instinct was that this feels like it's _probably_ fine, but could bite you later, but actually on experimenting, this pattern does have a potential bug in it!

This is potentially a subtle one, so apologies for the long comment incoming...

Note here that you're modifying the underlying query dictionary, which means that future requests that try to use `self.query` will _also_ have a `"name": "TEST"` field set, unless they actively reset it. That works out fine for your test, but is still at risk if the order or type of requests made to trello changes later.

In particular because we use this pattern elsewhere in this file, we need to be even more careful, since that code isn't always going to be interacted with in an identical order.

To reproduce an issue, try:
* Create a to-do called e.g. `Item 1`
* Create a second to-do, called e.g. `Item 2`
* Mark the **first** (`Item 1`) to-do as complete
* Note: it changed name to `Item 2` - we now have two `Item 2` entries!

The reason is that the `name` query has been left on the `params` dictionary from the second item creation, and so it gets passed through to Trello as part of the item update, and ends up overwriting the original name.

To resolve this then, we really want to avoid using the same query dictionary each time, although we still want to have to avoid repeating the same code. So instead of creating a single dictionary, tied to the class that we re-use, we could add a method that provides a copy of that dictionary fresh each time. For example:

```python
class TrelloItems:
    def __init__(self):
        self.url = "https://api.trello.com/1/"
        config = Config()
        self.board_id = config.TRELLO_BOARD_ID
        self.todo = config.TRELLO_LIST_ID_TODO
        self.done = config.TRELLO_LIST_ID_DONE
        self.config = config
        self.headers = {
            "Accept": "application/json"
        }

    def query(self):
        return {
            'key': self.config.TRELLO_API_KEY,
            'token': self.config.TRELLO_API_TOKEN
        }
```

And then everywhere that currently refers to `self.query` can instead create a fresh query with `query = self.query()`, for example in `change_card`:

```python
    def change_card(self, id):
        card = self.get_card_from_id(id)
        new_list = self.get_changed_list(card['list_id'])
        url = self.url + f"cards/{card['card_id']}"

        query = self.query()
        query['idList'] = new_list

        response = requests.put(
            url,
            headers=self.headers,
            params=query
        )
        return response.status_code
```

I notice this follows the pattern from the previous module, so apologies that I must have totally missed this in the previous review!